pages/createPlan/index.js

```diff
@@ -1,14 +1,9 @@
 import PlanForm from "@/components/PlanForm/PlanForm";
-import styled from "styled-components";
-
-const StyledFormContainer = styled.section`
-  min-height: 100vh;
-`;
 
 export default function CreatePlanView() {
   return (
-    <StyledFormContainer>
+    <section>
```

Comment:
I don't think that you need the `section` here either: the `form` can stand on its own and is semantic, so we don't need a wrapper, especially if we don't want to style it.